tests/integration/ambient/cni/main_test.go

```diff
@@ -210,6 +226,66 @@ func TestTrafficWithEstablishedPodsIfCNIMissing(t *testing.T) {
 		})
 }
 
+func TestCNIRestartsWithMissingKubeConfig(t *testing.T) {
```

Comment:
TODO in a follow up PR add a workflow which executes this test with IstioOwnedCNIConfig enabled

Reply:
It seems to be enabled in TestMain... And it seems to be skipped if owned=false.  What is missing?

Reply:
None of the test suites enable IstioOwnedCNIConfig so this test would always be skipped.